community/examples/slurm-gcp-v5-centos7.yaml

```diff
@@ -14,7 +14,7 @@
 
 ---
 
-blueprint_name: slurm-gcp-v5-cluster
+blueprint_name: slurm-gcp-v5-centos7
```

Comment:
Should this be "slurm-gcp-v5-hpc-centos7"? slurm-gcp also builds an image for the base centos7 image: https://github.com/SchedMD/slurm-gcp/blob/master/packer/example.pkrvars.hcl#L83

Reply:
good catch. I tried to reduce this as much as possible, as we have a limited number of characters, but I think the potential confusion justifies having the hpc in here.